product_sticker/views/product_sticker_views.xml

```diff
@@ -0,0 +1,299 @@
+                                name="product_category_id"
+                                options="{'no_create': True}"
+                            />
+                            <field
+                                name="product_attribute_id"
+                                options="{'no_create': True}"
+                                groups="product.group_product_variant"
+                            />
+                            <field
+                                name="product_attribute_value_id"
+                                options="{'no_create': True}"
+                                groups="product.group_product_variant"
+                            />
+                            <field name="show_sticker_note" widget="boolean_toggle" />
+                            <field
+                                name="note"
+                                colspan="2"
+                                nolabel="1"
+                                attrs="{'invisible': [('show_sticker_note', '=', False)]}"
+                                placeholder="Sticker note that goes below image"
+                            />
+                        </group>
+                        <group name="grp_image">
+                            <label for="image_1920" string="Image" colspan="2" />
+                            <field
+                                name="image_1920"
+                                widget="image"
+                                nolabel="1"
+                                colspan="2"
+                            />
+                            <div
+                                name="image_size"
+                                attrs="{'invisible': [('id', '=', False)]}"
+                                class="p-1 border-left border-top border-right border-bottom"
+                            >
+                                <field name="image_1920_size" invisible="1" />
+                                <i class="fa fa-info-circle text-black-50 pl-2 pr-2 " />
+                                <span
+                                    class="text-success font-italic font-weight-light"
+                                    attrs="{'invisible': [('image_1920_size', '&gt;', 30)]}"
+                                >Aceptable file size</span>
+                                <span
+                                    class="text-warning font-italic font-weight-light"
+                                    attrs="{'invisible': ['|', ('image_1920_size', '&lt;=', 30), ('image_1920_size', '&gt;', 60)]}"
+                                >Huge file size. The image should be optimized/reduced</span>
+                                <span
+                                    class="text-danger font-italic font-weight-light"
+                                    attrs="{'invisible': [('image_1920_size', '&lt;=', 60)]}"
+                                >Optimization required! Reduce the image size</span>
```

Comment:
Is this really needed?

When you use `image.mixin` (like you're doing), all you need to care about is the `image_1920` field. [Then you have 4 more sizes automatically produced](https://github.com/odoo/odoo/blob/19ff7a72eee6242e84e242ac0bdbf2d78a1be136/odoo/addons/base/models/image_mixin.py#L15-L19).

So, if you need to use a smaller sticker in the report, all you need to do is use `image_256` field or similar. I don't think we have to bother the user (and dev) about this detail that is solved by Odoo automatically.